picky-asn1-x509/src/private_key_info.rs

```diff
@@ -328,9 +360,30 @@ impl RsaPrivateKey {
     }
 }
 
+/// [Elliptic Curve Private Key Structure] (https://datatracker.ietf.org/doc/html/rfc5915#section-3)
+///
+/// EC private key information SHALL have ASN.1 type ECPrivateKey:
+///
+/// ```not_rust
+/// ECPrivateKey ::= SEQUENCE {
+///     version        INTEGER { ecPrivkeyVer1(1) } (ecPrivkeyVer1),
+///      privateKey     OCTET STRING,
+///      parameters [0] ECParameters {{ NamedCurve }} OPTIONAL,
+///      publicKey  [1] BIT STRING OPTIONAL
+///    }
+/// ```
+
+#[derive(Serialize, serde::Deserialize, Debug, Clone, PartialEq)]
```

Comment:
Just for uniformity with the rest of the file
```suggestion
#[derive(Serialize, Deserialize, Debug, Clone, PartialEq)]
```